packages/shared-backend/amplify/data/resource.ts

```diff
@@ -1,14 +1,33 @@
 import { a, defineData, type ClientSchema } from "@aws-amplify/backend";
 
 const schema = a.schema({
-  Todo: a
+  // User Table
+
+  // Resource Table
+  Resource: a
     .model({
-      content: a.string(),
+      resourceId: a.id().required(),
+      resourceType: a.enum(["Room", "Seat", "Equipment"]),
+      resourceSubtype: a.string(),
+      name: a.string().required(),
+      date: a.date().required(),
+      participants: a.string().array(),
+      reservations: a.hasMany("Reservation", "resourceId")
     })
+    .identifier(["resourceId"])
+    .secondaryIndexes((index) => [index("resourceType")])
+    // ex) resourceType 기준으로 쿼리
+    // const { data, errors } = await client.models.Resource.listResourceByResourceType({
+    //   resourceType: 'Seat',
+    // });
+
     .authorization((allow) => [allow.owner()]),
+
+  // Reservation Table
```

Comment:
```suggestion
  // Reservation Table
  
  Reservation: a
    .model({
       resourceId: a.id().required(),
       resource: a.belongsTo("Resource", "resourceId"), // 예약과 리소스의 관계
        })
```

이상태로샌드박스 돌리면 오류날거같아서 이거 추가해서 머지해주시면
제가 머지된거 받아서 다음 PR올리겠습니다압